flow.bkp/components-v2/surfaces/Collapse/index.js.flow

```diff
@@ -0,0 +1,20 @@
+// @flow
```

Comment:
I guess `components-v2` can be removed from bkp

Reply:
I wouldn't include the backup at all, we have version history to take care of that :)